_docs/performance-tuning/010-performance-tuning-introduction.md

```diff
@@ -3,9 +3,9 @@ title: "Performance Tuning Introduction"
 date:  
 parent: "Performance Tuning"
 ---
-You can apply performance tuning measures to improve how efficiently Drill queries data. To significantly improve performance in Drill, you must have knowledge about the underlying data and data sources, as well as familiarity with how Drill executes queries.
+You can change system options in Drill to improve the query performance. Before you improve performance in Drill, you must choose a layout of the data and the choose an appropriate file format specific to your use case. For example, for an analytic workload operating on historical time series data, then choosing Parquet as the file format and a partitioning scheme that uses time as a partitionining dimension would be a recommended approach. In the case you are directly querying data data sources, you need to have an understanding of the data source itself. Some familiarity with how Drill executes queries can also help. 
```

Comment:
"improve the query" --> "improve query"
"before you improve" --> "before you tune"
"series, then choosing" --> "series, choosing"
"would be a recommended" -> "is the recommended"

Actually, the whole sentence is too long. Maybe. "Suppose you had ... In this case, the recommended solution is..."

Reply:
I think it's okay for this introductory page to be vague and high-level.  Partitioning is discussed in more detail in its own child page.